packages/create-sitecore-jss/src/templates/node-headless-ssr-proxy/src/config.ts

```diff
@@ -15,11 +16,11 @@ const bundlePath = process.env.SITECORE_JSS_SERVER_BUNDLE || `./dist/${appName}/
 
 const serverBundle = require(bundlePath);
 
-httpAgents.setUpDefaultAgents(serverBundle);
+httpAgentsConfig.setUpDefaultAgents(serverBundle);
 
 const apiHost = process.env.SITECORE_API_HOST || 'http://my.sitecore.host';
 
-appName = appName || serverBundle.appName;
+appName = appName || (serverBundle.appName as string);
```

Comment:
You can add `appName` to the `ServerBundle` type
and on the line 17 - you can cast it as `ServerBundle`